subsys/mgmt/mcumgr/lib/README-zephyr.md

```diff
@@ -11,5 +11,4 @@ are described below:
 | `CONFIG_MCUMGR` | Enable the mcumgr management library. | n |
 | `CONFIG_MCUMGR_CMD_FS_MGMT` | Enable mcumgr handlers for file management | n |
 | `CONFIG_MCUMGR_CMD_IMG_MGMT` | Enable mcumgr handlers for image management | n |
-| `CONFIG_MCUMGR_CMD_LOG_MGMT` | Enable mcumgr handlers for log management | n |
```

Comment:
Need to squash this commit yet or change the commit description?
Leaving this commit may make the changes more readable.

Reply:
Sorry, you need to be more verbose because I did not get that.

Reply:
The Commit description for the commit containing this change says it will be squashed. You should squash this commit or remove that message in the commit description.
I would be in favor of leaving this commit to have better tracking of the changes.

Reply:
Removed information on squashing from commit messages.